doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,90 @@
+================
+Cross Validation
+================
+
+Cross validation allows to estimate an algorithm's performance on unseen data.
+
+It is based on training the model on a subset of the data, and testing on the
+remaining data.
```

Comment:
I propose being more precise here. All the available data is divided into train and test subsets. Then it is in fact the train subset which is further divided to perform cross-validation. That is, you train using part of the train subset and validate using another part of the train subset (we could say, the "validation subset"). This is of course still simplified, but it does make clear that the data used for validation is part of the train data. And more importantly, it is also clear that the test subset is never used for any training.